templates/noKibana/transactions.go

```diff
@@ -17,21 +17,103 @@ var Transactions = Object{
 			},
 		},
 	},
-
 	"mappings": Object{
 		"properties": Object{
-			"nonce": Object{
+			"data": Object{
+				"type": "keyword",
+			},
+			"esdtValues": Object{
+				"type": "keyword",
+			},
+			"fee": Object{
+				"index": "false",
+				"type":  "keyword",
+			},
+			"function": Object{
+				"type": "keyword",
+			},
+			"gasLimit": Object{
+				"index": "false",
+				"type":  "double",
```

Comment:
maybe use another type for gas price / limit / used? for the entire PR

Reply:
done.